modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-cluster-tasks-ref_{context}"]
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
```

Comment:
This {pipelines-shortnames} isn't being replaced properly, I'm assuming it's supposed to be singular?

```suggestion
An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
```